specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -0,0 +1,1949 @@
+                        },
+                        "compromisedEntity": {
+                            "readOnly": true,
+                            "type": "string"
+                        },
+                        "associatedResource": {
+                            "readOnly": true,
+                            "type": "string",
+                            "description": "Azure resource ID"
+                        },
+                        "extendedProperties": {
+                            "readOnly": true,
+                            "type": "object",
+                            "description": "Changing set of properties depending on alertName"
+                        },
+                        "systemSource": {
+                            "readOnly": true,
+                            "type": "string",
+                            "description": "The type of the alerted resource",
+                            "enum": [
+                                "azure",
+                                "non-azure"
+                            ]
+                        }
+                    }
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            }]
+        },
+        "DiscoveredSecuritySolutionList": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/DiscoveredSecuritySolution"
+                    }
+                }
+            }
+        },
+        "DiscoveredSecuritySolution": {
+            "type": "object",
+            "required": [
+                "properties",
+                "id",
+                "name",
+                "type",
+                "location"
+            ],
+            "properties": {
+                "properties": {
+                    "required": [
+                        "securityFamily",
+                        "offer",
+                        "publisher",
+                        "sku"
+                    ],
+                    "type": "object",
+                    "properties": {
+                        "securityFamily": {
+                            "$ref": "#/definitions/securityFamily"
+                        },
+                        "offer": {
+                            "type": "string",
+                            "description": "The security solutions' image offer"
+                        },
+                        "publisher": {
+                            "type": "string",
+                            "description": "The security solutions' image publisher"
+                        },
+                        "sku": {
+                            "type": "string",
+                            "description": "The security solutions' image sku"
+                        }
+                    }
+                },
+                "type": {

```

Comment:
it should also have `location` property.

Reply:
Added